setup.py

```diff
@@ -1,16 +1,30 @@
 """ Setup script for mir_datasets. """
 from setuptools import setup, find_packages
 
+with open('README.md', 'r') as fdesc:
+    long_description = fdesc.read()
+
 if __name__ == '__main__':
     setup(
         name='mirdata',
         version='0.1.1',
-        description='Common loaders for MIR datasets.',
+        description='Data management utilities for reproducible research in music information retrieval (MIR).',
```

Comment:
I got used to the "common loaders for reproducible...", but this works for me too

Reply:
It's not clear what "common loaders" mean. "Common"= "widespread" or "common"="compatible"?

Reply:
@magdalenafuentes note that @rabitt approved this change, so if you're OK with this `description` string, we can resolve the conversation and merge this PR.
This is not an irreversible design decision, so it's fine to GTD and fine-tune later.